design/cli-install-changes.md

```diff
@@ -0,0 +1,286 @@
+      --pod-cpu-request string                     CPU request for Velero pod. A value of "0" is treated as unbounded. Optional. (default "500m")
+      --pod-mem-limit string                       memory limit for Velero pod. A value of "0" is treated as unbounded. Optional. (default "256Mi")
+      --pod-mem-request string                     memory request for Velero pod. A value of "0" is treated as unbounded. Optional. (default "128Mi")
+      --client-burst int                           maximum number of requests by the server to the Kubernetes API in a short period of time (default 30)
+      --client-qps float32                         maximum number of requests per second by the server to the Kubernetes API once the burst limit has been reached (default 20)
+      --default-backup-ttl duration                how long to wait by default before backups can be garbage collected (default 720h0m0s)
+      --disable-controllers strings                list of controllers to disable on startup. Valid values are backup,backup-sync,schedule,gc,backup-deletion,restore,download-request,restic-repo,server-status-request
+  -h, --help                                       help for server
+      --log-format                                 the format for log output. Valid values are text, json. (default text)
+      --log-level                                  the level at which to log. Valid values are debug, info, warning, error, fatal, panic. (default info)
+      --metrics-address string                     the address to expose prometheus metrics (default ":8085")
+      --plugin-dir string                          directory containing Velero plugins (default "/plugins")
+      --profiler-address string                    the address to expose the pprof profiler (default "localhost:6060")
+      --restore-only                               run in a mode where only restores are allowed; backups, schedules, and garbage-collection are all disabled. DEPRECATED: this flag will be removed in v2.0. Use read-only backup storage locations instead.
+      --restore-resource-priorities strings        desired order of resource restores; any resource not in the list will be restored alphabetically after the prioritized resources (default [namespaces,storageclasses,persistentvolumes,persistentvolumeclaims,secrets,configmaps,serviceaccounts,limitranges,pods,replicaset,customresourcedefinitions])
+      --terminating-resource-timeout duration      how long to wait on persistent volumes and namespaces to terminate during a restore before timing out (default 10m0s)
+
+```
+
+2) `velero backup-location`
+Commands/flags for backup locations. 
+
+```
+     set 
+        --default string                                  sets the default backup storage location (default "default") (NEW, -- was `server --default-backup-storage-location) 
```

Comment:
Where will this be stored? Will it set the `server --default-backup-storage-location` flag, or an annotation on the BSL itself?

Reply:
Yes, good thinking ahead. Storing it as part of the BSL spec would be nice as an idea, since it sounds like it's an attribute of it. It would also be nice to, when we do a `get`, have a column that displays which one is the default. What I'd like to avoid is when querying for that needing to load all BSLs and iterate thru them. If we can add a label and annotation to the BSL I think we could do that?